project-p2-layouts-master/code/about.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>Responsive Website Project</title>
+    <link rel="stylesheet" href="./about.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie-edge">
+</head>
+<body>
+    <header>
+        <div class="hamburger-dropdown-menu">
+          <input type="checkbox" class="toggler">
+          <div class="hamburger"><div></div></div>
+          <div class="menu">
+            <div>
+              <div>
+                <ul>
+                <li><a href="./index.html">HOME</a></li>
+                <li><a href="./about.html">ABOUT</a></li>
+                <li><a href="./essays.html">ESSAYS</a></li>
+                <li><a href="./letters.html">LETTERS</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
Remember indentation - check out [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) to learn more.

```
<div class="hamburger-dropdown-menu">
  <input type="checkbox" class="toggler">
  <div class="hamburger">
    <div></div>
  </div>
  <div class="menu">
    <div>
      <div>
        <ul>
          <li>
            <a href="./index.html">HOME</a>
          </li>
          <li>
            <a href="./about.html">ABOUT</a>
          </li>
          <li>
            <a href="./essays.html">ESSAYS</a>
          </li>
          <li>
            <a href="./letters.html">LETTERS</a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
```